pantheon-bundle/frontend/src/app/moduleDisplay.tsx

```diff
@@ -335,6 +360,31 @@ class ModuleDisplay extends Component<any, any, any> {
         }
     }
 
+    private fetchAttributesFilePath = async (data) => {
+        await this.getVariantParam()
+        const path = data.location.pathname.substring(PathPrefixes.MODULE_PATH_PREFIX.length)
+        // console.log("[fetchAttributesFilePath] path =>", path)
+        let repo = ''
+        const group = path.split("/")
+        repo = group[2]
+        fetch('/content/repositories/' + repo + '/module_variants/' + this.state.variant + '.harray.json')
+            .then((response) => {
+                if (response.ok) {
+                    return response.json()
+                } else {
+                    throw new Error(response.statusText)
+                }
+            })
+            .then(responseJSON => {
+                if (responseJSON["pant:attributesFilePath"] !== undefined) {
+                    this.setState({ attributesFilePath: responseJSON["pant:attributesFilePath"] })
+                }
+            })
+            .catch((error) => {
+                console.log(error)
+            })
+    }
+
```

Comment:
Tagging this at the bottom, but this entire file looks substantially similar to the assemblyDisplay.tsx file. Can we combine them to reduce duplication?

Reply:
I agree we can use a shared component for assemblyDisplay and moduleDisplay. I would like it to be a separate ticket.  This is just the initial work of introducing assembly on the UI.

Reply:
Ok, we'll agree that this is tech debt and we'll bring in this PR as-is.